ksqldb-engine/src/test/java/io/confluent/ksql/function/UdfLoaderTest.java

```diff
@@ -187,6 +190,45 @@ public void shouldLoadDecimalUdfs() {
     assertThat(fun.name().text(), equalToIgnoringCase("floor"));
   }
 
+  @Test
+  public void shouldLoadLambdaReduceUdfs() {
+    // Given:
+    final SqlLambda schema =
+        SqlLambda.of(
+            ImmutableList.of(SqlTypes.INTEGER, SqlTypes.INTEGER, SqlTypes.INTEGER),
+            SqlTypes.INTEGER);
+
+    // When:
+    final KsqlScalarFunction fun = FUNC_REG.getUdfFactory(FunctionName.of("reduce_map"))
```

Comment:
nit: Let's just use some random generic function names for these tests

Reply:
I think you have to use an existing function name so it can find it in the function library, unless there's a way to mock that? When I changed it to `lambda_func` I got `io.confluent.ksql.util.KsqlException: Can't find any functions with the name 'lambda_func'`

Reply:
Oh really? That seems really odd since we'd break these tests if we changed the function name. It feels like we should have these functions mocked. If that's how it is then leaving this is fine then.